src/isomorphic/children/ReactChildren.js

```diff
@@ -164,7 +353,7 @@ function mapChildren(children, func, context) {
   return result;
 }
 
-function forEachSingleChildDummy(traverseContext, child, name) {
```

Comment:
We're already importing the `emptyFunction` library in `ReactChildren`, could use just use `emptyFunction.thatReturnsNull` and get rid of this?